tests/ia/alerts_test.js

```diff
@@ -57,20 +61,47 @@ BeforeSuite(async ({
     testEmail,
   );
 
-  ruleIdForEmailCheck = await rulesAPI.createAlertRule({
+  // Preparation steps for checking Alert via webhook server
+  // eslint-disable-next-line no-template-curly-in-string
+  await I.verifyCommand('bash -x ${PWD}/testdata/ia/gencerts.sh');
+  await I.verifyCommand('docker-compose -f docker-compose-webhook.yml up -d');
```

Comment:
Do you need a wait right after starting the container?

Reply:
there is no need to wait as we have like 2 mins before the alert will be triggered